### PR TITLE
command-line arguments option, supply multiple inputs at once, can convert images outside working directory

### PR DESCRIPTION
**Pull request by Kuuhhl**
I modified the code to allow for the following:
* Supplying multiple image paths with command-line arguments / with the TUI
* Support for images outside the script's directory

### CODE REVIEW

**Comment by paulopacitti** [comment]
Hey @Kuuhhl, thanks for submitting a pull request 💟 I've just merged @Gabulhas 's PR that implements command line arguments (But just one image per execution). Could you resolve the conflicts and check how to merge @Gabulhas and yours solution?

**Comment by Kuuhhl** [comment]
Conflicts are resolved.

**Comment by paulopacitti** [comment]
Reviewing it 🔍

File: factory/__main__.py
Comment (paulopacitti) [inline factory/__main__.py]: Could you add something like "Which image(s) do you want to manufacture? (images paths separated by spaces)"?

**Review by paulopacitti — CHANGES_REQUESTED** [review]
Nice work! If you could just change that string, I'll merge right after it 😉

File: factory/__main__.py
Comment (Kuuhhl) [reply factory/__main__.py]: Sure! The prompt is changed now.

**Comment by paulopacitti** [comment]
I'll update `README.md` with the multiple arguments feature and release a new version soon. Thank you so much!

**Pull request merged**
